<?xml version="1.0" encoding="utf-8"?>
<launch>
	<node name="inference_bts" pkg="inference_bts" type="inference_bts" output="screen">
		<param name="imgTopic" value="/front_60degree_camera/compressed" />
		<param name="modelWeightName" value="$(find inference_bts)/data/bts_outdoor.pt" />
	</node>
</launch>
